docs/index.md

```diff
@@ -138,10 +139,11 @@ Or, to include the optional HTTP/2 support, use:
 $ pip install httpx[http2]
 ```
 
-To include the optional brotli decoder support, use:
+To include the optional brotli and zstandard decoder support, use:
 
 ```shell
 $ pip install httpx[brotli]
+$ pip install httpx[zstd]
```

Comment:
I think it would be better to merge those 2 into a single command. New developers sometimes don't know how to merge multiple optional deps and the index.md lacks such an example. Also when using "and" it indicates the presence of both decoders, while running those 2 commands sequentially will end up with just 1 decoder (2nd command will uninstall brotli).

```suggestion
To include the optional brotli and zstandard decoders support, use:

```shell
$ pip install httpx[brotli,zstd]
```